tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -1024,6 +1113,36 @@ static void test_device_pb_remote_server_unproved(void)
 	PASS();
 }
 
+/** @brief A device running a Remote Provisioning server that is used to provision unprovisioned
```

Comment:
Why do you need Remote Provisioning server on this device?

Reply:
it's because of this: https://github.com/zephyrproject-rtos/zephyr-bt-sig/issues/390#issue-1684912093 but as explained in https://github.com/zephyrproject-rtos/zephyr-bt-sig/issues/390#issuecomment-1550943754, you are right, we do not need that

Reply:
I didn't get why imposter model is required? Could you elaborate?

Reply:
to highjack RPR PDU Send command. Before it procedure is normal, provisioning link is opened. After RPR server receives first PDU Send device becomes unresponsive, and RPR Client on 1st device will timeout in 60s.

Reply:
This is somewhat similar to what happens in MBT bsim tests, when imposter model makes message to be lost. Here we additionally suspend whole mesh on device with RPR Server, so there is no more communication. This is basically only reliable way to trigger RPR timeout of 60s on RPR Server. In other case, lower timeout will be reached on RPR Server and 2nd device will send link report to RPR Client, closing provisioning link prematurely